src/web/components/HydrateOnce.tsx

```diff
@@ -22,12 +22,16 @@ const isReady = (dependencies: unknown[]): boolean => {
  * @param {number} index - Used when there are multiple elements the same (eg. RichLinks)
  * @param {Array} waitFor - An array of variables that must be defined before the task is executed
```

Comment:
Need to update these docs. I'm not sure what `name` is

Reply:
`name` is just me trying to preserve the performance output so that we log out the same information as before, eg. `hydrate-youtube`. I think I still need to run through and pass these strings in. I wonder if there's a better way of doing this, like grabbing the component name

Reply:
I've gone with removing this `name` prop altogether. We were using it to log individual hydration times to the console but we don't do anything more with it yet so there's not really a use case here that justifies the extra complexity that wuold be needed to continue to support using a different log name for each hydration step